src/validation.cairo

```diff
@@ -206,9 +208,22 @@ fn validate_coinbase(block: @Block, total_fees: u256) -> Result<(), ByteArray> {
     Result::Ok(())
 }
 
+// Return BTC reward => pow to 8 to transform into Sats. Otherwise difficult to cast to u64 correctly after if needed
+fn compute_block_reward(block_height: u32) -> Result<u64, ByteArray> {
```

Comment:
No need to return a Result.

Reply:
I changed it to return a u64 only, thank you